install/deploy/templates/mev-boost.tmpl

```diff
@@ -19,6 +19,7 @@ services:
       - NETWORK={{.Network}}
       - MEV_BOOST_PORT={{.MevBoost.Port}}
       - MEV_BOOST_RELAYS={{.MevBoost.GetRelayString}}
+      - MEV_BOOST_ADDITIONAL_FLAGS={{.MevBoost.AdditionalFlags}}
```

Comment:
I don't see this being added to the config struct in this pr. I'm assuming it was already cherry-picked in to v2?

Reply:
I think it's already being passed, unless I'm misunderstanding. [Code Link](https://github.com/rocket-pool/smartnode/blob/v2/shared/config/mev-boost-config.go#L73)

Also looks like it gets passed [here](https://github.com/rocket-pool/smartnode/blob/v2/shared/config/mev-boost-config.go#L282). I also checked the UI on my node and confirmed the parameter box is present but it won't be used without this change.